modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GPAD_conf.pm

```diff
@@ -45,38 +46,37 @@ sub default_options {
 	## General parameters
     'registry'      => $self->o('registry'),   
     'release'       => $self->o('release'),
-    'pipeline_name' => 'gpad_loader',       
+    'pipeline_name' => 'gpad_loader_'.$self->o('release'),
     'email'         => $self->o('ENV', 'USER').'@ebi.ac.uk',
-    'output_dir'    => '/nfs/nobackup/ensemblgenomes/'.$self->o('ENV', 'USER').'/workspace/'.$self->o('pipeline_name'),     
+    'output_dir'    => '/nfs/nobackup/ensembl/'.$self->o('ENV', 'USER').'/workspace/'.$self->o('pipeline_name'),
 
     ## Location of GPAD files
-    'gpad_directory' => [],
+    'gpad_directory' => '',
 
 	## Email Report subject
-    'email_subject'       	   => $self->o('pipeline_name').' GPAD loading pipeline has finished',
+    'email_subject'  => $self->o('pipeline_name').' GPAD loading pipeline has finished',
 
     ## Remove existing GO annotations
     # on '1' by default
-    'delete_existing' => 1,
+    'delete_existing_go' => 1,
```

Comment:
I think it's clearer to have a single 'delete_existing' parameter; if you delete the analysis, you need to delete the associated GO terms anyway, to prevent orphans. And there's no benefit to keeping the analysis if you've deleted all its GO terms.

Reply:
Sure, that make sense. I changed it to avoid confusion but what you say make total sense.

Reply:
see latest commit.